cfme/infrastructure/cluster.py

```diff
@@ -122,9 +123,13 @@ def get_all_clusters(do_not_navigate=False):
     """Returns list of all clusters"""
     if not do_not_navigate:
         sel.force_navigate('infrastructure_clusters')
-    clusters = set([])
+    cluster = namedtuple('cluster', ['name', 'id'])
+    clusters = set()
+    title_path = "//div[@id='quadicon']/../../../tr/td/a[contains(@href,'cluster/show')]"
     for page in paginator.pages():
-        for title in sel.elements(
-                "//div[@id='quadicon']/../../../tr/td/a[contains(@href,'cluster/show')]"):
-            clusters.add(sel.get_attribute(title, "title"))
+        for title in sel.elements(title_path):
```

Comment:
Previously we've had a Quadicon.all() in places like this, can this work here too?